app/src/main/java/com/lastaoutdoor/lasta/models/user/Language.kt

```diff
@@ -12,4 +12,12 @@ enum class Language(@StringRes val languageText: Int) {
   fun resourcesToString(context: Context): String {
     return context.getString(this.languageText)
   }
+
+  fun toLocale(): String {
+    return when (this) {
+      ENGLISH -> "en"
+      FRENCH -> "fr"
+      GERMAN -> "de"

```

Comment:
It would be nice to test for each language in case for one the toLocale would be incorrect :)